staging/publishing/rules.yaml

```diff
@@ -26,7 +26,11 @@ rules:
       branch: release-1.19
       dir: staging/src/k8s.io/code-generator
     name: release-1.19
-    go: 1.15.5
+    go: 1.15.8
+  - source:
+      branch: release-1.20
+      dir: staging/src/k8s.io/code-generator
+    name: release-1.20
```

Comment:
the added sources in this file don't look right... bad pick?

Reply:
yep, will remove

Reply:
done